dialect.go

```diff
@@ -47,12 +49,12 @@ type PostgresDialect struct{}
 
 func (pg PostgresDialect) createVersionTableSQL() string {
 	return `CREATE TABLE goose_db_version (
-            	id serial NOT NULL,
-                version_id bigint NOT NULL,
-                is_applied boolean NOT NULL,
-                tstamp timestamp NULL default now(),
-                PRIMARY KEY(id)
-            );`
+				id serial NOT NULL,
+				version_id bigint NOT NULL,
+				is_applied boolean NOT NULL,
+				tstamp timestamp NULL default now(),
+				PRIMARY KEY(id)
+			);`
```

Comment:
Is this improving readability/formatting? Would be nice, to have a separate PR for this.